.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+on:
+  pull_request:
+  push:
+    branches: main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.stack
+            ./.stack-work
+          key: ${{ runner.os }}-${{ hashFiles('stack.yaml') }}-${{ hashFiles('*.cabal') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ hashFiles('stack.yaml') }}-
+            ${{ runner.os }}-
+      - uses: freckle/stack-action@697bdc2715b64b3d0aa3f12406f7eef3ca3c9735
```

Comment:
Is there a reason you depending on a specific sha instead of a version tag, or just `@main`?

Reply:
Yep, none of the tags support the `weeder: false` option and I generally think of shas as being safer than branches (e.g. due to moving target). Though since we own the action and this package it probably makes sense to embrace `main`. Done in latest commit!

Reply:
Oh yeah, we just need to release a new tag since adding `weeder`. Feel free to do so!